### DISCUSSION TITLE
Версия 2.0 beta 8

### DISCUSSION THREAD

**Discussion by lirix360** [lirix360/ReadmangaGrabber#50] [Announcements]
Новые функции версии 2.0:

- Веб-GUI для скачивания манги. Работает в любом современном браузере.

Изменения beta 7:

- Добавлена возможность выбора предпочитаемого перевода на ReadManga.
- Ассеты (css, js, шрифты) встроены в приложение вместо загрузки со сторонних площадок.
- Багфиксы.

Сообщать об ошибках и предложениях [можно в обсуждениях](https://github.com/lirix360/ReadmangaGrabber/discussions).

**Бета-версия доступна только для Windows.**

<hr /><em>This discussion was created from the release <a href='https://github.com/lirix360/ReadmangaGrabber/releases/tag/v2.0beta8'>Версия 2.0 beta 8</a>.</em>

**Comment by ghost** [comment]
ураааа

**Comment by Vonter-T-800** [comment]
Можно ли скачивать сразу большой ПДФ файл?

**Reply by Vonter-T-800** [reply]
т.е. сборник из глав сразу в пдф

**Reply by lirix360** [reply]
Может быть добавлю такой функционал в будущих версиях.

**Comment by Meloni4** [comment]
просто грузит и не выдает глав

**Reply by lirix360** [reply]
А можно подробнее?
Какую мангу, с какого сайта?

**Reply by Meloni4** [reply]
Синяя тюрьма: Блю лок

**Comment by DressCodeBoy** [comment]
на повтор бы неудавшиеся главы с ошибками, а то на одной ошибку при декодировании выдал и минус вся глава
![image](https://user-images.githubusercontent.com/52357422/170803334-c27cf793-9e12-41d1-aec1-28a28c3ad4bd.png)
